ui/ceval/src/stockfishProtocol.js

```diff
@@ -1,11 +1,17 @@
 var m = require('mithril');
 
+var EVAL_REGEX = new RegExp(''
+  + /^info depth (\d+) seldepth \d+ multipv (\d+) /.source
+  + /score (cp|mate) ([-\d]+) /.source
+  + /(?:(upper|upper)bound )?nodes (\d+) nps (\d+) /.source
+  + /(?:hashfull \d+ )?tbhits \d+ time (\d+) /.source
+  + /pv ?(.*)/.source);
```

Comment:
Regarding the `?`: If `pv` is there, then there should also always be at least one move.